default.json5

```diff
@@ -44,184 +44,165 @@
   ],
   packageRules: [
     {
-      matchPackagePatterns: [
-        "^io\\.confluent[.:]",
-        "^org\\.apache\\.kafka[.:]",
+      matchPackageNames: [
+        "io.confluent:*",
+        "org.apache.kafka:*",
       ],
       registryUrls: [
         "https://packages.confluent.io/maven",
       ],
     },
     {
-      matchPackagePrefixes: [
-        "com.hivemq.commons:",
+      matchPackageNames: [
+        "com.hivemq.commons:*",
       ],
       registryUrls: [
         "https://maven.pkg.github.com/hivemq/hivemq-commons",
       ],
     },
     {
-      matchPackagePrefixes: [
-        "com.byteowls:",
+      matchPackageNames: [
+        "com.byteowls:*",
       ],
       registryUrls: [
         "https://maven.pkg.github.com/hivemq/vaadin-chartjs",
       ],
     },
     {
-      matchPackagePatterns: [
-        "^com\\.github\\.everit-org\\.json-schema[.:]",
-      ],
-      registryUrls: [
-        "https://jitpack.io",
-      ],
-    },
-    {
-      groupName: "all major dependencies",
-      groupSlug: "all-major",
       matchUpdateTypes: [
-        "major",
+        "minor",
       ],
-    },
-    {
+      matchCurrentVersion: "!/^0/",
       groupName: "all minor dependencies",
       groupSlug: "all-minor",
-      matchUpdateTypes: [
-        "minor",
-      ],
     },
     {
-      groupName: "all patch dependencies",
-      groupSlug: "all-patch",
       matchUpdateTypes: [
         "patch",
       ],
       matchCurrentVersion: "!/^0/",
+      groupName: "all patch dependencies",
+      groupSlug: "all-patch",
       automerge: true,
       platformAutomerge: false,
     },
     {
       matchManagers: [
         "npm",
       ],
-      groupName: "npm all major dependencies",
-      groupSlug: "all-major-npm",
-      matchUpdateTypes: [
-        "major",
-      ],
-    },
-    {
-      matchManagers: [
-        "npm",
-      ],
-      groupName: "npm all minor dependencies",
-      groupSlug: "all-minor-npm",
       matchUpdateTypes: [
         "minor",
       ],
+      matchCurrentVersion: "!/^0/",
+      groupName: "all minor npm dependencies",
+      groupSlug: "all-minor-npm",
```

Comment:
Just a reminder: whenever we change the `groupName/groupSlug`, Renovate will create new PRs and will not close the old ones.